### PR TITLE
Show iteration test status

### PR DESCRIPTION
**Pull request by kntsoriano**

### CODE REVIEW

File: app/javascript/components/mentoring/discussion/IterationHeader.tsx [hunk 1]
```diff
@@ -23,6 +40,7 @@ export const IterationHeader = ({
             Submitted {fromNow(iteration.createdAt)}
           </time>
         </div>
+        <p>{testsStatus}</p>
```
Comment (kntsoriano) [inline app/javascript/components/mentoring/discussion/IterationHeader.tsx:42]: @iHiD I'm rendering it as a string for now.

File: app/javascript/components/mentoring/discussion/IterationHeader.tsx [hunk 2]
```diff
@@ -10,6 +11,22 @@ export const IterationHeader = ({
   iteration: Iteration
   latest: boolean
 }): JSX.Element => {
+  const [testsStatus, setTestsStatus] = useState(iteration.testsStatus)
+
+  useEffect(() => {
+    const iterationChannel = new IterationChannel(
+      iteration.uuid,
+      (iteration) => {
+        console.log('boom')
```
Comment (SleeplessByte) [inline app/javascript/components/mentoring/discussion/IterationHeader.tsx]: 💥

**Review by SleeplessByte — APPROVED** [review]

File: app/channels/iteration_channel.rb [hunk 3]
```diff
@@ -1,7 +1,9 @@
 class IterationChannel < ApplicationCable::Channel
   def subscribed
     # Assert that the user owns this iteration
-    iteration = current_user.iterations.find_by!(uuid: params[:uuid])
+    iteration = Iteration.find_by!(uuid: params[:uuid])
+
+    return unless iteration.viewable_by?(current_user)
```
Comment (ErikSchierboom) [inline app/channels/iteration_channel.rb]: Is there any reasonable way in which a non-owner subscribes to this channel? In other words, should we somehow log this as an error?

File: app/javascript/components/mentoring/discussion/IterationHeader.tsx (hunk 2 above)
Comment (ErikSchierboom) [reply app/javascript/components/mentoring/discussion/IterationHeader.tsx]: 🧯

**Review by ErikSchierboom — APPROVED** [review]

File: app/channels/iteration_channel.rb (hunk 3 above)
Comment (SleeplessByte) [reply app/channels/iteration_channel.rb]: If it's not expected, an error would be logical, yes.

File: app/javascript/components/mentoring/discussion/IterationHeader.tsx (hunk 2 above)
Comment (iHiD) [reply app/javascript/components/mentoring/discussion/IterationHeader.tsx]: @kntsoriano Can you add a `TODO` to these please to work out how we acutally deal with them later?

File: app/channels/iteration_channel.rb (hunk 3 above)
Comment (iHiD) [reply app/channels/iteration_channel.rb]: @kntsoriano can you add a TODO to raise a bugsnag in this scenario pls.

**Review by iHiD — APPROVED** [review]

**Review by iHiD — APPROVED** [review]

File: app/javascript/components/mentoring/discussion/IterationHeader.tsx (hunk 2 above)
Comment (kntsoriano) [reply app/javascript/components/mentoring/discussion/IterationHeader.tsx]: Hahahahaha!

**Pull request merged**